src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/AssessmentController.kt

```diff
@@ -64,10 +62,15 @@ class AssessmentController(
     perPage: Int?,
   ): ResponseEntity<List<AssessmentSummary>> {
     val user = userService.getUserForRequest()
+    val manuallyFilter = xServiceName == ServiceName.temporaryAccommodation
 
     val summaries = when {
       xServiceName == ServiceName.temporaryAccommodation && crn != null -> assessmentService.getAssessmentSummariesByCrnForUser(user, crn, xServiceName)
-      else -> assessmentService.getVisibleAssessmentSummariesForUser(user, xServiceName)
+      xServiceName == ServiceName.temporaryAccommodation -> assessmentService.getVisibleAssessmentSummariesForUserCAS3(user)
+      else -> {
+        val domainSummaryStatuses = statuses?.map { assessmentTransformer.transformApiStatusToDomainSummaryState(it) } ?: emptyList()
+        assessmentService.getVisibleAssessmentSummariesForUserCAS1(user, domainSummaryStatuses)
+      }
```

Comment:
Having this change being to the `else` branch might change the behaviour for CAS2, so we should make sure that this won't cause them problems.

@edavey, @Harriethw: does CAS2 use this assessment logic?

Reply:
I think the previous behaviour here was to return an emptyList() for CAS2 (this was returned by the assessmentSrvice function i've changed), now it will return approved premises assessments, which was not an intentional change on my part, so good spot! i'll add an explicit CAS2 branch in here to retain existing behaviour

Reply:
We don't have assessments in CAS2 so we aren't ever calling this atm

Reply:
Thanks @Harriethw, in that case i'll update the code to throw an unsupported operation exception for CAS2 to make it clear going forward 👍